src/snowflake/snowpark/modin/plugin/_internal/index.py

```diff
@@ -996,15 +1074,13 @@ def get_indexer_for(self, target: Any) -> Any:
 
         Examples
         --------
+        # Snowpark pandas converts NaN to None
```

Comment:
Nit:
```suggestion
        # Snowpark pandas converts np.nan, pd.NA, pd.NaT and None to None
```

Reply:
you can leave out the "None to None" part haha